### ISSUE TITLE
Add handling for HTTP 429 responses

### ISSUE THREAD

**Issue by cuu508** [bdd/runitor#66]
When pinging a check via slug, if the request gets rate-limited, Healthchecks.io can return HTTP 429 response: https://healthchecks.io/docs/http_api/#success-slug

When this happens, runitor outputs:

```
2022/10/17 13:15:06 Ping(start): max tries reached after try 3. last error: <nil>
```

It would be nice if runitor would print a more descriptive "rate limit exceeded" message here. 

Note: this only happens when pinging by slug. When pinging by uuid, Healthchecks.io always returns HTTP 200, even in rate-limited cases, for backwards compatibility.

Note 2: Healthchecks.io (the hosted service) has rate-limiting, but the open-source Healthchecks project does not. For efficiency, the rate-limiting is done by nginx, so there's no sign of it in the python code.

**Comment by bdd** [comment]
This was a bug.
Added the commit above. Will cut a release soon.

**Comment by bdd** [comment]
I released [v1.0.0](https://github.com/bdd/runitor/releases/tag/v1.0.0) :tada: including this bugfix.
